core/metrics/application.py

```diff
@@ -103,92 +103,92 @@ def calculate_summarized_application_metrics(app):
 
 
 # Alternative calculation method, drilling down per-version rather than per-application.
-def calculate_detailed_application_metrics(application, interval=rrule.MONTHLY):
```

Comment:
Again, what does this have to do with "machine validation"? Changes for another PR, perhaps?